File: algorithm/453. Minimum Moves to Equal Array Elements/453.py
class Solution(object):
    def minMoves(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums)==0:
            return 0
        nums.sort()
        result,min=0,nums[0]
        for i in range(len(nums)):
            result+=nums[i]-min
        return result

File: algorithm/453. Minimum Moves to Equal Array Elements/453.cpp
class Solution {
public:
    int minMoves(vector<int>& nums) {
        if(nums.size()==0) return 0;
        sort(nums.begin(),nums.end());
        int min=nums[0],result=0;
        for(int i=0; i<nums.size(); i++)
        {
            result+=(nums[i]-min);
        }
        return result;
    }
};
